kotlintest-assertions/src/main/kotlin/io/kotlintest/matchers/matchers.kt

```diff
@@ -10,7 +10,7 @@ fun withClue(clue: String, thunk: () -> Any) {
   try {
     thunk()
   } catch (e: AssertionError) {
-    throw Failures.failure("$clue $e")
+    throw Failures.failure("$clue ${e.message}")
```

Comment:
I suppose there is no test for this at the moment. Maybe we should create one for this behavior as well?

Reply:
Strangely there are no tests in the `kotlintest-assertions` module at all (not even a test source path). I'm happy creating a test, any preference on where I should create it? Should I just add the test sources to the assertions module?

Reply:
There's a special source-set for testing at the moment: `kotlintest-tests`. They're all there. I'm not sure why all tests are in the same place at the moment, but that's what we have atm :P

Reply:
The reason is because in order to test kotlintest as a genuine junit-platform plugin, it needs to be tested as a jar on the dependency path. In order to do this, kotlintest-core must be built before the tests, hence why the test are in a separate package.

The ones for assertions _could_ live in assertions though, it's just the "core" tests that _need_ be separate. Arrow tests for example are in the arrow module.

Reply:
OK I just added tests. As an aside, the tests were in the package `com.sksamuel.kotlintest.matchers` which struck me as odd (I'm guessing due to historical reasons)

Reply:
The reason is that there are certain tests that only work if they're not in the io.kotlintest package. So I just put them all in my private domain.